src/m365/aad/commands/user/user-recyclebinitem-restore.ts

```diff
@@ -0,0 +1,84 @@
+import { User } from '@microsoft/microsoft-graph-types';
+import { Logger } from '../../../../cli/Logger';
+import GlobalOptions from '../../../../GlobalOptions';
+import request, { CliRequestOptions } from '../../../../request';
+import { validation } from '../../../../utils/validation';
+import GraphCommand from '../../../base/GraphCommand';
+import commands from '../../commands';
+
+interface CommandArgs {
+  options: Options;
+}
+
+interface Options extends GlobalOptions {
+  id: string;
+}
+
+class AadUserRecycleBinItemRestoreCommand extends GraphCommand {
+  public get name(): string {
+    return commands.USER_RECYCLEBINITEM_RESTORE;
+  }
+
+  public get description(): string {
+    return 'Restores a user from the recycle bin in the current tenant';
```

Comment:
Please align the description from the docs with this one. I would remove the wording "current tenant"